homeassistant/components/analytics_insights/config_flow.py

```diff
@@ -55,15 +56,20 @@ async def async_step_user(
         """Handle the initial step."""
         errors: dict[str, str] = {}
         if user_input is not None:
-            if not user_input.get(CONF_TRACKED_INTEGRATIONS) and not user_input.get(
-                CONF_TRACKED_CUSTOM_INTEGRATIONS
+            if all(
+                [
+                    not user_input.get(CONF_TRACKED_ADDONS),
+                    not user_input.get(CONF_TRACKED_INTEGRATIONS),
+                    not user_input.get(CONF_TRACKED_CUSTOM_INTEGRATIONS),
+                ]
             ):
                 errors["base"] = "no_integrations_selected"
             else:
                 return self.async_create_entry(
                     title="Home Assistant Analytics Insights",
                     data={},
                     options={
+                        CONF_TRACKED_ADDONS: user_input.get(CONF_TRACKED_ADDONS, []),
```

Comment:
i've decided to just use `get` here, instead of implementing a complex config entry migration logic